src/main/java/com/google/jenkins/plugins/storage/GoogleCloudStorageUploader.java

```diff
@@ -162,11 +168,12 @@ public String getDisplayName() {
     }
 
     /**
-     * @return the default uploads when the user configure
-     *         {@link GoogleCloudStorageUploader} for the first time.
+     * @return the default uploads when the user configure {@link
+     * GoogleCloudStorageUploader} for the first time.
```

Comment:
I'm not sure how `@link` deals with a asterisks inside them. It's probably best just to put the whole tag on the next line.